evap/results/templates/results_course_detail_questionnaires.html

```diff
@@ -29,16 +29,11 @@
                     <td>
                         {% if result.show and result.count > 0 %}
                             <div class="distribution-bar twipsify"
-                                title="{% for answer, rel in result.distribution.items %}{{answer|likertname}}: {{rel|floatformat}}%{% if not forloop.last %}, {% endif %}{% endfor %}"
-                                data-twipsy="<div style='display:table;'>{% for answer, rel in result.distribution.items %}<div style='display:table-row;'><div style='display:table-cell; text-align: right;'>{{answer|likertname}}:&nbsp;</div><div style='display:table-cell; text-align: right;'>{{rel|floatformat}}%</div></div>{% endfor %}</div>">
+                                title="{% for answer, rel in result.distribution.items %}{% ifequal result.question.kind "L" %}{{answer|likertname}}{% else %}{{answer}}{% endifequal %}: {{rel|floatformat}}%{% if not forloop.last %}, {% endif %}{% endfor %}">
```

Comment:
could you bring some line breaks into this? also, i'd prefer `if a == b`to `ifequal a b`.